analyses/molecular-subtyping-ATRT/01-ATRT-molecular-subtyping-data-prep.Rmd

```diff
@@ -0,0 +1,391 @@
+```
+
+### CN data
+
+```{r}
+# Collapse target overexpressed gene vectors
+tyr_genes <- paste(tyr_genes, collapse = "|")
+shh_genes <- paste(shh_genes, collapse = "|")
+myc_genes <- paste(myc_genes, collapse = "|")
+
+# Join focal CN data with metadata
+cn_metadata <- cn_df %>%
+  dplyr::left_join(metadata,
+                   by = c("biospecimen_id" = "Kids_First_Biospecimen_ID")) %>%
+  dplyr::select(gene_symbol,
+                sample_id,
+                Kids_First_Participant_ID,
+                biospecimen_id,
+                status) %>%
+  dplyr::filter(sample_id %in% atrt_expression_df$sample_id) %>%
+  dplyr::mutate(
+    SMARCB1_focal_status = dplyr::case_when(gene_symbol == "SMARCB1" ~ status,
+                                            TRUE ~ "neutral"),
+    SMARCA4_focal_status = dplyr::case_when(gene_symbol == "SMARCA4" ~ status,
+                                            TRUE ~ "neutral"),
+    Overexpressed_gene_sets = dplyr::case_when(
+      stringr::str_detect(gene_symbol, tyr_genes) &
+        status != "loss" ~ "TYR_genes",
+      stringr::str_detect(gene_symbol, shh_genes) &
+        status != "loss" ~ "SHH_genes",
+      stringr::str_detect(gene_symbol, myc_genes) &
+        status != "loss" ~ "MYC_genes",
+      TRUE ~ "NA"
+    )
+  ) %>%
+  dplyr::select(-gene_symbol) %>%
+  dplyr::distinct()
+
+#Display `cn_metadata`
+cn_metadata %>%
+  head(n = 15) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+
+# Join ATRT expression data with focal CN data
+atrt_expression_cn_df <- atrt_expression_df %>%
+  dplyr::left_join(cn_metadata, by = "sample_id") %>%
+  dplyr::select(-status)
+```
+
+### ssGSEA data
+
+```{r}
+# Calculate ssGSEA mean and sd
+ssGSEA_means <- rowMeans(ssGSEA, na.rm = TRUE)
+ssGSEA_sd <- apply(ssGSEA, 1, sd, na.rm = TRUE)
+
+# Subtract mean
+ssGSEA_zscored <- sweep(ssGSEA, 1, ssGSEA_means, FUN = "-")
+
+# Divide by SD remove NAs and Inf values from zscore for genes with 0 in normData
+ssGSEA_zscored <- sweep(ssGSEA_zscored, 1, ssGSEA_sd, FUN = "/") %>% 
+  dplyr::na_if(Inf) %>% 
+  na.omit()
+
+# Transpose
+transposed_ssGSEA <- t(ssGSEA_zscored)
+
+# Select wanted pathways and merge metadata 
+transposed_ssGSEA <- transposed_ssGSEA %>%
+  as.data.frame() %>%
+  tibble::rownames_to_column("Kids_First_Biospecimen_ID") %>%
+  dplyr::select(
+    Kids_First_Biospecimen_ID,
+    HALLMARK_MYC_TARGETS_V1,
+    HALLMARK_MYC_TARGETS_V2,
+    HALLMARK_NOTCH_SIGNALING
+  ) %>%
+  dplyr::left_join(select_metadata, by = "Kids_First_Biospecimen_ID")
+
+#Display `transposed_ssGSEA`
+transposed_ssGSEA %>%
+  head(n = 15) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+
+# Join ATRT expression and focal CN data with transposed ssGSEA data
+atrt_expression_cn_df <- atrt_expression_cn_df %>%
+  dplyr::left_join(transposed_ssGSEA,
+                    by = "sample_id")
+```
+
+### Tumor mutation burden data
+
+```{r}
+# Join tumor mutuation data with metadata
+tmb_df <- tmb_df %>%
```

Comment:
Are you using the subset files here by any chance? Is that why all of `tmb` is `NA` in the final table?

Reply:
Regarding this point, you can delete the `data/snv-consensus_11122019` folder you have locally and rerun `bash download-data.sh` or you could check out the branch over on https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/295 which allows you to overwrite the testing files I suspect you have locally.

Reply:
You are correct.